dev-libs/aws-sdk-cpp/aws-sdk-cpp-1.7.178.ebuild

```diff
@@ -0,0 +1,147 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit cmake-utils python-single-r1
+
+DESCRIPTION="AWS SDK for C++"
+HOMEPAGE="https://aws.amazon.com/sdk-for-cpp/"
+SRC_URI="https://github.com/aws/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+MODULES=(
+	access-management apigateway backup batch budgets chime cloud-desktop cloud-dev cloud-media cloud-mobile
+	cloudformation cloudfront cloudhsm cloudsearch cloudwatch cognito config dlm ec2 ecr ecs eks elasticache
+	elasticbeanstalk elasticloadbalancing elasticmapreduce email es events fsx globalaccelerator iot kinesis kms
+	lambda lex license-manager lightsail machinelearning macie managedblockchain marketplace opsworks organizations
+	other personalize polly queues rds rekognition resource-groups route53 s3 sagemaker secretsmanager securityhub
+	serverlessrepo shield sns sqs textract transcribe translate waf
+)
+
+IUSE="+http +rtti +ssl static-libs test unity-build ${MODULES[*]}"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="
+	http?	( net-misc/curl:= )
+	polly?	( media-sound/pulseaudio )
+	ssl?	( dev-libs/openssl:0 )
```

Comment:
And here. Any idea about libressl compatibility? openssl has a soname, so if this gets linked during compilation (its in DEPEND after all) you need to add a subslot binder, `:=`.

Reply:
The build system, as far as I can tell, only detects that it is there and uses it as a default provider for SSL.

This might imply that the SDK *could* be compatible to libressl, but I honestly do not know.

However, I have enabled support for it and if a user with a libressl system finds out that the assumption is wrong, it can be patched to make it work or the support can be removed.